src/main/kotlin/com/figure/gradle/semver/internal/valuesources/GitCalculateSemverValueSource.kt

```diff
@@ -129,19 +131,29 @@ internal abstract class GitCalculateSemverValueSource : ValueSource<String, GitC
                 initialConfig.withBranchMatchingConfig(versionStrategy.get())
             }
 
-            git.hasBranch(GitRef.Branch.DEVELOP.name).isNotEmpty() -> {
-                log.semverInfo("Enabling Git Flow mode")
-                initialConfig.withBranchMatchingConfig(flowVersionCalculatorStrategy(versionModifier.get()))
+            git.hasBranch(GitRef.Branch.DEVELOP) && git.hasBranch(GitRef.Branch.MAIN) -> {
```

Comment:
I understand the detection approach and order, but it is worth documenting in the README since this is not configurable.